### ISSUE TITLE
upverter, fritzing, & other design program templates

### ISSUE THREAD

**Issue by jywarren** [oshwa/certification-mark#4]
Similarly to #1, we'd love template contributions!

**Comment by justind000** [comment]
I use upverter and just tried making a new part. Upverter doesn't allow footprints to be made using pictures, so I don't think a typical footprint is going to be possible. I ended up installing Inkscape and created my own little logo. The font as described in the readme comes pre-installed with Inkscape. 

I have attached the svg I created; it should be easy enough for someone to edit it with Inkscape to change the UID. 
[us0069.svg.zip](https://github.com/oshwa/certification-mark/files/1150723/us0069.svg.zip)

**Comment by jywarren** [comment]
Hi, just looking at it here: 

![screenshot 2017-07-16 at 3 46 00 pm](https://user-images.githubusercontent.com/24359/28250733-ee2c9afa-6a3d-11e7-97aa-e632b4fd2ced.png)

it could use a little more space between the digits and the box above. See for example:

https://github.com/oshwa/certification-mark/raw/master/artwork/example.png

But sorry, i don't use Upverter -- can you upload an SVG? Thanks for your work on this!

**Comment by justind000** [comment]
Hello,
Sorry for the delay, but I have attached a zip file in #19 with a slightly changed logo with the extra spacing along with a very short writeup of how to change it further within Inkscape. 

As for Upverter, you can upload a png and place it on the silkscreen layer to achieve the 'footprint'. I had some issues with using svg files.

**Comment by justind000** [comment]
I also just threw this together for:

https://oshw-svg-logo.herokuapp.com/?serial=US####

It's just a quick hack of an svg to replace 'serial' in the URL with what you pass to it. 

For example,
https://oshw-svg-logo.herokuapp.com/?serial=US0001 

Gives you a certification mark with US0001. It will take a serial number between 4 and 6 numerical digits plus the two letters in front.

It's running on heroku for free. Anyone is welcome to use the source and improve it, I'm not much of a web developer.

**Comment by NdK73** [comment]
I tried converting the official SVG to KiCAD footprint, but the result was frankly quite ugly.
So I used it as a template for this:
```
(module OSHW (layer F.Cu) (tedit 5A379F9A)
  (fp_text reference G*** (at -3.048 -6.096) (layer F.SilkS) hide
    (effects (font (thickness 0.3)))
  )
  (fp_text value ITcertno (at 0 3.556) (layer F.SilkS)
    (effects (font (thickness 0.3)))
  )
  (fp_line (start 3.683 0) (end 4.572 -5.715) (layer F.SilkS) (width 0.35))
  (fp_text user OSHW (at -0.127 -1.397) (layer F.SilkS)
    (effects (font (size 3.2 2.2) (thickness 0.35)))
  )
  (fp_line (start 3.556 -4.699) (end -5.334 -4.699) (layer F.SilkS) (width 0.55))
  (fp_line (start -5.334 -4.699) (end -5.334 1.524) (layer F.SilkS) (width 0.55))
  (fp_line (start -5.334 1.524) (end 5.461 1.524) (layer F.SilkS) (width 0.55))
  (fp_line (start 5.461 1.524) (end 5.461 -4.953) (layer F.SilkS) (width 0.55))
)
```
The result is like this:
![oshw-kicad_mod](https://user-images.githubusercontent.com/1850337/34103196-27df9a2c-e3ec-11e7-88e4-28acb215cace.png)

Is it Ok to use? I could use the same system to create a schematic symbol, too...

**Comment by mwweinberg** [comment]
That looks good to me

**Comment by jywarren** [comment]
Indeed! Could you open a pull request with this to be included into the library of templates? What's the best way to share it?

**Comment by NdK73** [comment]
Just after cloning this repo, I found it already contains way better "footprints" for KiCAD. Sorry for the noise.

**Comment by jywarren** [comment]
Oh, I'm sorry! I don't use it myself so I hadn't remembered that. Thanks in
any case!
